packages/web-components/src/accordion/accordion.stories.ts

```diff
@@ -0,0 +1,121 @@
+import { html } from '@microsoft/fast-element';
+import type { Args, Meta } from '@storybook/html';
+import { renderComponent } from '../__test__/helpers.js';
+import { AccordionItemExpandIconPosition, AccordionItemSize } from '../accordion-item/accordion-item.options.js';
+import type { Accordion as FluentAccordion } from './accordion.js';
+import './define';
+import '../accordion-item/define';
+
+type AccordionStoryArgs = Args & FluentAccordion;
+type AccordionStoryMeta = Meta<AccordionStoryArgs>;
+
+const storyTemplate = html<AccordionStoryArgs>`
+  <fluent-accordion expand-mode=${x => x.expandMode}>
+    <fluent-accordion-item
+      size=${x => x.size}
+      heading-level=${x => x.headingLevel}
+      expandIconPosition=${x => x.expandIconPosition}
```

Comment:
```suggestion
      expand-icon-position=${x => x.expandIconPosition}
```
Everywhere in the stories.